pages/project/read/views/components/rops.jsx

```diff
@@ -124,20 +113,24 @@ export function Rops({ project = {}, ropsYears = [], url, today = new Date() })
 
   const rops = project.rops;
 
-  const startDate = `${thisYear}-01-01`;
   const endDate = project.revocationDate || project.expiryDate;
-  const ropNotRequired = endDate && isBefore(new Date(endDate), new Date(startDate));
```

Comment:
This was the cause of the primary bug. Projects that ended prior to 2022-01-01 were not showing _any_ ROP related content.